pkg/scheduler/framework/interface.go

```diff
@@ -307,10 +307,16 @@ type QueueSortPlugin interface {
 }
 
 // EnqueueExtensions is an optional interface that plugins can implement to efficiently
-// move unschedulable Pods in internal scheduling queues.
+// move unschedulable Pods in internal scheduling queues. Only plugins
+// impacting scheduling will take effect, other implemented plugins are useless and
+// harmless.
```

Comment:
not quite clear to me, do we have any plugin doesn't impact scheduling at all? also, I cannot understand this statement 
> other implemented plugins are useless and harmless.

Reply:
sorry about the ambiguous words，I updated my comments right now. 🥲